theme/components/sections/home/banner.html.twig

```diff
@@ -1,9 +1,10 @@
-{% set bannerSrc = image.src|asset_url %}
+{% set bannerSrc = image.src is defined ? image.src|asset_url : 'images/home-banner.png'|asset_url %}
```

Comment:
Trying this paradigm out, mark the schema as `optional` and provide the defaults in the template